src/login/oidc/oidcHandlers/LegacyImplicitFlowOidcHandler.ts

```diff
@@ -50,6 +50,7 @@ export default class LegacyImplicitFlowOidcHandler implements IOidcHandler {
   }
 
   async handle(oidcLoginOptions: IOidcOptions): Promise<ISolidSession> {
+    oidcLoginOptions = oidcLoginOptions as IAccessTokenOidcOptions;
```

Comment:
As with my other comment: it'd be great if you could find a way to avoid this cast.

```suggestion
  async handle(oidcLoginOptions: IAccessTokenOidcOptions): Promise<ISolidSession> {
```